src/components/LandingPage/Footer.tsx

```
@@ -0,0 +1,75 @@
+import handMiddleFinger from '@/assets/illustrations/hand-middle-finger.svg'
+
+const Footer = () => {
+    return (
+        <footer className="relative flex h-52 items-end justify-between bg-black px-8 pb-10 md:items-center md:px-20 md:pb-0">
+            <section className="flex flex-col gap-1">
+                <div className="flex">
+                    <Image src={PEANUT_LOGO} alt="Peanut Logo" width={110} height={40} />
+                </div>
+                <p className="text-xs text-white">
+                    made with love by{' '}
+                    <a className="underline" href="https://squirrellabs.dev/" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)